src/Testing.java

```diff
@@ -21,5 +20,22 @@ public void testWenjingManalAndAddSeeds()
 
 		}
 			
+		/*@Test
```

Comment:
why is this test commented out?  the initial seeds should probably be 0, but the test should either be fixed or deleted.